### PR TITLE
Edit and re-format VDS -> VAT doc [VS-821]

### PR DESCRIPTION
**Pull request by rsasch**

### CODE REVIEW

File: scripts/variantstore/variant_annotations_table/README.md [hunk 1]
```diff
@@ -1,96 +1,49 @@
...
-* VDS -- this can be created using the GVS callset creation pipeline
-* Ancestry TSV -- this can be created using the Ancestry pipeline (and this is also the first input needed to run the WDL)
-We recommend using a Terra Python Notebook with a small Hail cluster to run the Python script and make these VAT pipeline inputs.
-Once those inputs have been created, they can be used by the GvsCreateVATfromVDS WDL
+Once you have copied them, run `python3 hail_create_vat_inputs.py` with the following input(s)
 
-  These are the required parameters which must be supplied to the GvsCreateVATfromVDS workflow:
+* `--ancestry_file` GCS pointer to the TSV file that maps `sample_name`s to sub-populations
+* `--vds_path` GCS pointer to the top-level direcotory of the VDS
+* `--sites_only_vcf` (optional, should be pre-populated by the `GvsExtractAvroFilesForHail` workflow) the location to write the sites-only VCF to; save this for the `GvsCreateVATfromVDS` workflow
```
Comment (RoriCremer) [inline scripts/variantstore/variant_annotations_table/README.md:22]: ooooh I didn't realize this was optional...interesting

**Review by RoriCremer — APPROVED** [review]
THANK YOU!

File: scripts/variantstore/variant_annotations_table/GvsValidateVAT.wdl [hunk 2]
```diff
@@ -4,113 +4,113 @@ import "../wdl/GvsUtils.wdl" as Utils
 
 workflow GvsValidateVat {
     input {
-        String query_project_id
+        String project_id
```
Comment (gbggrant) [inline scripts/variantstore/variant_annotations_table/GvsValidateVAT.wdl:7]: Grrr - I thought I had got them all!!

File: scripts/variantstore/variant_annotations_table/README.md [hunk 3]
```diff
@@ -1,96 +1,49 @@
...
-**GvsValidateVAT** checks and validates the created VAT and prints a report of any failing validation
-
-
-### Run GvsCreateVATfromVDS:
+### The VAT pipeline is a python script and a set of WDLs
 
-Most of the inputs are constants — like the reference, or a table schema — and don't require any additional work, or have defaults set in the WDL). 
-However for the specific data being put in the VAT, two inputs need to be created.
+- hail_create_vat_inputs.py is a script to be run in a notebook terminal to generate a sites-only VCF.
+- [GvsCreateVATfromVDS.wdl](/scripts/variantstore/wdl/GvsCreateVATfromVDS.wdl) creates the variant annotations table from a sites only VCF and an acestry file TSV.
```
Comment (gbggrant) [inline scripts/variantstore/variant_annotations_table/README.md:6]:
```suggestion
- [GvsCreateVATfromVDS.wdl](/scripts/variantstore/wdl/GvsCreateVATfromVDS.wdl) creates the variant annotations table from a sites only VCF and an ancestry file TSV.
```

File: scripts/variantstore/variant_annotations_table/README.md [hunk 4]
```diff
@@ -1,96 +1,49 @@
...
-This Python script needs the following inputs:
-* VDS -- this can be created using the GVS callset creation pipeline
-* Ancestry TSV -- this can be created using the Ancestry pipeline (and this is also the first input needed to run the WDL)
-We recommend using a Terra Python Notebook with a small Hail cluster to run the Python script and make these VAT pipeline inputs.
-Once those inputs have been created, they can be used by the GvsCreateVATfromVDS WDL
+Once you have copied them, run `python3 hail_create_vat_inputs.py` with the following input(s)
 
-  These are the required parameters which must be supplied to the GvsCreateVATfromVDS workflow:
+* `--ancestry_file` GCS pointer to the TSV file that maps `sample_name`s to sub-populations
+* `--vds_path` GCS pointer to the top-level direcotory of the VDS
```
Comment (gbggrant) [inline scripts/variantstore/variant_annotations_table/README.md:21]:
```suggestion
* `--vds_path` GCS pointer to the top-level directory of the VDS
```

**Review by gbggrant — APPROVED** [review]
Looks good!

**Pull request merged**